packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -102,6 +132,16 @@ class MethodWebsocketRequestHandler {
       );
     }
 
+    if (endpointMethodConnector is! MethodStreamConnector) {
+      server.serverpod.logVerbose(
+        'Endpoint method is not a valid stream method: $message',
+      );
+      return OpenMethodStreamResponse.buildMessage(
+        connectionId: message.connectionId,
+        responseType: OpenMethodStreamResponseType.endpointNotFound,
+      );
+    }
```

Comment:
It can only be either a MethodStreamConnector or a MethodConnector (aka MethodCallConnector).  My thinking was that the error message is a bit different that could be helpful for debugging? For example if you remove all streams from a method it will be converted to a MethodCallConnector but you might still have clients trying to connect to it.